moxy/org.eclipse.persistence.moxy/src/main/java/org/eclipse/persistence/internal/jaxb/XMLJavaTypeConverter.java

```diff
@@ -51,12 +48,12 @@
 public class XMLJavaTypeConverter extends org.eclipse.persistence.oxm.mappings.converters.XMLConverterAdapter {
     protected Class<?> boundType = Object.class;
     protected Class<?> valueType = Object.class;
-    protected Class<?> xmlAdapterClass;
+    protected Class<XmlAdapter<?,?>> xmlAdapterClass;
```

Comment:
this should be `Class<? extends XmlAdapter<?,?>>` instead.

Reply:
Yes, makes sense. Will fix with next iteration.

Reply:
Removed.